### ISSUE TITLE
[Class hierarchy] Fibroblasts are not epithelial cells!

### ISSUE THREAD

**Issue by kiri93** [obophenotype/cell-ontology#1186]
**CL term**
In the current ontology, the term `CL_2000096 ` (reticular layer of dermis fibroblast) has a `"is_a"` relation with `CL:0000066` (epithelial cells). This is, as far as I understand biology, inaccurate. 

**Suggested revision and reasons**
[I tried to fix it in a Pull request](https://github.com/obophenotype/cell-ontology/pull/1182) and I was told to modify the `src/ontology/cl-edit.owl` file directly. Now, the corresponding entry in this file looks like this:
```
# Class: obo:CL_2000096 (reticular layer of dermis fibroblast)

AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:TermGenie"^^xsd:string) obo:IAO_0000115 obo:CL_2000096 "Any fibroblast that is part of a reticular layer of dermis."^^xsd:string)
AnnotationAssertion(oboInOwl:created_by obo:CL_2000096 "TermGenie"^^xsd:string)
AnnotationAssertion(oboInOwl:creation_date obo:CL_2000096 "2015-08-11T16:28:16Z"^^xsd:string)
AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_2000096 "cell"^^xsd:string)
AnnotationAssertion(oboInOwl:id obo:CL_2000096 "CL:2000096"^^xsd:string)
AnnotationAssertion(rdfs:label obo:CL_2000096 "reticular layer of dermis fibroblast"^^xsd:string)
EquivalentClasses(obo:CL_2000096 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001993)))
```

No traces of `CL:0000066` (epithelial cells). I speculate that the term `obo:UBERON_0001993` (reticular layer of dermis) might be problematic, since it has a `is part of` relation with `UBERON:0001993` (epidermal-dermal junction). But I'm at the limit of my understanding of these things; I'm not an ontologist.

**Comment by dosumis** [comment]
Hi @kiri93 - many thanks for looking.  I've assigned @shawntanzk who is working on a more general fix.  Many thanks again for spotting the problem and attempting edits.

**Comment by shawntanzk** [comment]
whoops slippery fingers with the thumbs down reaction, meant to be thumbs up >.< 
Anyway @kiri93 think this is fixed with the changes made in #1185 
reticular layer of dermis fibroblast was placed as an epithelial cell due to reasoning, hence you did not see it in the edit file. We have relaxed the equivalent class and it is now solved (at least once that PR goes through) 
![Screenshot 2021-07-20 at 22 47 38](https://user-images.githubusercontent.com/76212760/126399771-52b97cbb-bad1-408f-9b26-d2591b039f09.png)
I've also made a changed to fibroblast of the reticular layer of dermis - I added a "the" cause I think it's clearer that way, but happy to change back if it's a matter of consistency etc.
